src/commands/import-resolver.ts

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+import {SnowpackConfig} from '../config';
+import {findMatchingMountScript} from '../util';
+import {isDirectoryImport} from './build-util';
+import srcFileExtensionMapping from './src-file-extension-mapping';
+
+interface ImportResolverOptions {
+  fileLoc: string;
+  dependencyImportMap: any;
+  isBuild: boolean;
+  isBundled: boolean;
+  config: SnowpackConfig;
+}
+
+/**
+ * Create a import resolver function, which converts any import relative to the given file at "fileLoc"
+ * to a proper URL. Returns false if no matching import was found, which usually indicates a package
+ * not found in the import map.
+ */
+export function createImportResolver({
+  fileLoc,
+  dependencyImportMap,
+  isBuild,
+  isBundled,
+  config,
+}: ImportResolverOptions) {
+  const webModulesScript = config.scripts.find((script) => script.id === 'mount:web_modules');
+  const webModulesLoc = webModulesScript ? webModulesScript.args.toUrl : '/web_modules';
+
+  return function importResolver(spec: string): string | false {
+    if (spec.startsWith('http')) {
```

Comment:
nit-picking here, but are we sure this won’t catch an npm that starts with `http`?

Reply:
...no ...no we are not